pre-docs/writing_plans.md

```diff
@@ -71,9 +68,9 @@ should be run on.
 
 ```
 plan mymodule::my_plan(
-  String[1]                             $load_balancer,
-  Variant[String[1], Array[String[1]]]  $frontends,
-  Variant[String[1], Array[String[1]]]  $backends,
+  String[1]   $load_balancer,
+  TargetSpec  $frontends,
```

Comment:
I think the Puppet guidelines would suggest a single space after TargetSpec, not two.